systems/robotInterfaces/CMakeLists.txt

```diff
@@ -15,5 +33,11 @@ if (MATLAB_FOUND)
     pods_use_pkg_config_packages(octomapWrapper octomap)
 
   endif()
-
+  
+  if (LCM_FOUND)
+    add_mex(constructQPLocomotionPlanmex constructQPLocomotionPlanmex.cpp)
+    target_link_libraries(constructQPLocomotionPlanmex drakeQPLocomotionPlan drakeUtil)
+  endif()
 endif()
+
+add_subdirectory(@QPLocomotionPlanCPPWrapper)
```

Comment:
i would have only added the subdirectory if matlab and lcm are found, and kept the cmakelists inside the directory cleaner.